rust/datafusion-examples/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+[package]
+name = "datafusion-examples"
```

Comment:
FYI @andygrove  -- I am not sure what you think about possibly releasing the examples as a crate of their own. I personally think it would be necessary. However perhaps it would be the right way to to eventually to somehow link to the example code from the docs.rs page https://docs.rs/datafusion/3.0.0/datafusion/

I filed this JIRA https://issues.apache.org/jira/browse/ARROW-11863 to track that

Reply:
I'd lean on not publishing by default, so I'd prefer a `publish = false` on the Cargo file. I support clarifying this as part of ARROW-11863 @alamb.

Reply:
Added `publish = false`.
The examples are currently also not on https://docs.rs/datafusion/3.0.0/datafusion/ I think - agree that it would be very useful.

Reply:
> The ok, outcome would be to point the docs from docs.rs always at master

another would be to link at a versioned url (based on published version)
https://github.com/apache/arrow/tree/apache-arrow-3.0.0